test/expected/loader.out

```diff
@@ -301,7 +301,7 @@ ERROR:  Mismatched timescaledb version. Shared object file mock-4-mismatch, SQL
 \set ON_ERROR_STOP 0
 --should not allow since the errored-out mock-4 above already poisoned the well.
 CREATE EXTENSION timescaledb VERSION 'mock-5';
-ERROR:  Cannot create the extension after the another version has already been loaded
+ERROR:  extension "timescaledb" already exists
```

Comment:
Also this error message seems misleading. TimescaleDB does not exist, but the library has been loaded.